libs/form-builder/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "form-builder",
+  "version": "0.1.0",
+  "description": "Form builder which is used to create forms with the FormIO builder.",
+  "scripts": {
+    "lint": "eslint . --ext .ts,.tsx",
+    "test": "vitest --passWithNoTests",
```

Comment:
Do we expect this lib to ever have unit tests? If not, we can get rid of the whole `vitest` part.

Reply:
Since we don't now, lets remove it.